subsystems/armSubsystem.py

```diff
@@ -0,0 +1,126 @@
+import wpilib
+import rev
+import constants 
+import commands2
+import phoenix5
+
+class ArmSubsystem(commands2.Subsystem):
+    def __init__(self) -> None:
+        
+        self.armmotor1 = rev.CANSparkMax(constants.ELEC.arm_motor1_CAN_ID, rev.CANSparkMax.MotorType.kBrushless)
+        self.armmotor2 = rev.CANSparkMax(constants.ELEC.arm_motor2_CAN_ID, rev.CANSparkMax.MotorType.kBrushless)
+        self.armmotor2.setInverted(True)
+        self.armLimitSwitch = wpilib.DigitalInput(constants.ELEC.arm_limit_switch)
+        self.motorgroup = wpilib.MotorControllerGroup(self.armmotor1, self.armmotor2)
+        
+        # self.tofSensor = playingwithfusion.TimeOfFlight(0) # Create a tofSensor Instance
+        # self.tofSensor.RangingMode(1) # Set the range to medium
+        
+    #     # Get both encoder values for the arm and later display them
+    #     self.encoder1 = self.armmotor1.getEncoder()
+    #     self.encoder2 = self.armmotor2.getEncoder()
+        
+    #     # Initialize PID controller for spark max
```

Comment:
There's a ton of PID-related code here, which is all commented out, but which you probably don't just want to delete.  You could put it on a different branch.  But the branch that gets merged into `main` shouldn't have all this debris lying around.